src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Tests/Fixtures/DataSetMetaGeneratorExtensions.cs

```diff
@@ -0,0 +1,115 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Tests.Fixtures;
+
+public static class DataSetMetaGeneratorExtensions
+{
+    public static Generator<DataSetMeta> DefaultDataSetMeta(this DataFixture fixture)
+        => fixture.Generator<DataSetMeta>().WithDefaults();
+
+    public static Generator<DataSetMeta> DefaultDataSetMeta(
+        this DataFixture fixture,
+        int filters,
+        int indicators,
+        int locations,
+        int timePeriods,
+        int maxFilterOptions = 10,
+        int maxLocationOptions = 10)
+    {
+        var locationMeta = fixture.DefaultLocationMeta()
+            .WithOptions(() => fixture.DefaultLocationOptionMeta().GenerateRandomList(maxLocationOptions))
+            .GenerateList(locations);
```

Comment:
Might as well put this inline below like with the filters. Or, assign the filters, indiciators and geographic levels a variable too to be consistent? Don't feel strongly either way though

Reply:
We've deliberately set this as separate variable because we want to only generate a single list of location meta. We're then using this twice, once in `WithGeographicLevels` and `WithLocations`.

If we inlined it, we'd end up generating two lists that are potentially different and would  mean the geographic levels and locations aren't synced up properly.

Reply:
didn't realise, it's good where it is in that case :)